wazimap_ng/points/views.py

```diff
@@ -211,3 +215,44 @@ def list(self, request, profile_id, geography_code):
         except ObjectDoesNotExist as e:
             logger.exception(e)
             raise Http404
+
+
+class LocationListByDistance(generics.ListAPIView):
+    queryset = models.Location.objects.all().annotate(icon=F('category__profilecategory__theme__icon'),
+                                                      theme_id=F('category__profilecategory__theme__id'),
+                                                      theme_name=F('category__profilecategory__theme__name'),
```

Comment:
Should be formatted like rest of the code

Reply:
done